contracts/base-staking/Staking.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "./CompoundRateKeeperV2.sol";
+import "./interfaces/IStaking.sol";
+
+contract Staking is IStaking, CompoundRateKeeperV2 {
```

Comment:
I agree with @Hrom131 that this contract not a development module and should be moved to a different repo